tests/test_api.py

```diff
@@ -30,7 +31,8 @@ def setUp(self):
         self.api = API(self.server, self.username, self.password, str(self.totp.now()))
         for i in range(3):
             try:
-                self.api.authenticate()
+                result = self.api.authenticate()
+                self.assertTrue(result is True)
```

Comment:
@rmol This is a setUp call, please do not add any `assert` statement here, if it fails, the actual test to catch that is `test_api_auth`. You will find that call does not use any decorator.

Reply:
Yep. Thanks.